spring-cloud-dataflow-docs/src/main/asciidoc/tasks.adoc

```diff
@@ -360,3 +360,212 @@ Another option to start a task using the `task-launcher` would be to create a st
+applications that comprises our composed task as well as the composed task
+definition.  We also see that each of the generated
+names for the child tasks is comprised of the name of the composed task and
+the name of the application separated by a dash `-`.  i.e. _my-composed-task_ `-`
+_mytaskapp_.
+
+===== Task Application Parameters
+The task applications that comprise the composed task definition can also
+contain parameters.  For example:
+```
+dataflow:> task create my-composed-task --definition "mytaskapp --displayMessage=hello && timestamp --format=YYYY"
+```
+
+==== Launching a Composed Task
+Launching a composed task is done the same way as launching a stand-alone task.
+i.e.
+```
+task launch my-composed-task
+```
+Once the task is launched and assuming all the tasks complete successfully you will
+see three task executions when executing a `task execution list`.  For example:
+```
+dataflow:>task execution list
+╔══════════════════════════╤═══╤════════════════════════════╤════════════════════════════╤═════════╗
+║        Task Name         │ID │         Start Time         │          End Time          │Exit Code║
+╠══════════════════════════╪═══╪════════════════════════════╪════════════════════════════╪═════════╣
+║my-composed-task-timestamp│713│Wed Apr 12 16:43:07 EDT 2017│Wed Apr 12 16:43:07 EDT 2017│0        ║
+║my-composed-task-mytaskapp│712│Wed Apr 12 16:42:57 EDT 2017│Wed Apr 12 16:42:57 EDT 2017│0        ║
+║my-composed-task          │711│Wed Apr 12 16:42:55 EDT 2017│Wed Apr 12 16:43:15 EDT 2017│0        ║
+╚══════════════════════════╧═══╧════════════════════════════╧════════════════════════════╧═════════╝
+```
+In the example above we see that my-compose-task launched and it also launched
+the other tasks in sequential order.
+
+==== Destroying a Composed Task
+The same command used to destroy a stand-alone task is the same as destroying a
+composed task.  The only difference is that destroying a composed task will
+also destroy the child tasks associated with it.   For example
+
+```
+dataflow:>task list
+╔══════════════════════════╤═══════════════════════════════════════════════════════════════
+║        Task Name         │                      Task Definition
+╠══════════════════════════╪═══════════════════════════════════════════════════════════════
+║my-composed-task          │mytaskapp && timestamp
+║my-composed-task-mytaskapp│mytaskapp
+║my-composed-task-timestamp│timestamp
+
+...
+dataflow:>task destroy my-composed-task
+dataflow:>task list
+╔══════════════════════════╤═══════════════════════════════════════════════════════════════
+║        Task Name         │                      Task Definition
+╠══════════════════════════╪═══════════════════════════════════════════════════════════════
+╚══════════════════════════╧═══════════════════════════════════════════════════════════════
+```
+
+=== Composed Task DSL
+
+==== Conditional Execution
+Conditional execution is expressed using a double ampersand symbol &&.
```

Comment:
For consistency, let's wrap ampersand with ticks (eg., `&&`). Other operators in the following sections are formatted correctly.